modules/ocf_mail/templates/site_vhost/mysql-alias-map.erb

```diff
@@ -0,0 +1,8 @@
+hosts = mysql
+user = ocfmail-ro
```

Comment:
Could some duplication be removed between this template and the `mysql-alias-domains` template since they have the same content except for the SQL query? Maybe the query could be another variable that they each get?